gulpfile.js

```diff
@@ -0,0 +1,208 @@
+const _ = require('lodash');
+const argv = require('yargs').argv;
+const backstopjs = require('backstopjs');
+const clean = require('gulp-clean');
+const colors = require('ansi-colors');
+const execSync = require('child_process').execSync;
+const file = require('gulp-file');
+const fs = require('fs');
+const gulp = require('gulp');
+const notify = require('gulp-notify');
+const path = require('path');
+const puppeteer = require('puppeteer');
+const PluginError = require('plugin-error');
+
+const BACKSTOP_DIR = 'backstop_data';
+const CONFIG_TPL = { 'url': 'http://%{site-host}', 'root': '%{path-to-site-root}' };
+const FILES = {
+  siteConfig: path.join(BACKSTOP_DIR, 'site-config.json'),
+  temp: path.join(BACKSTOP_DIR, 'backstop.temp.json'),
+  tpl: path.join(BACKSTOP_DIR, 'backstop.tpl.json')
+};
+
+['reference', 'test', 'openReport', 'approve'].map(action => {
+  gulp.task('backstopjs:' + action, () => {
+    return runBackstopJS(action);
+  });
+});
+
+/**
+ * Returns the list of the scenarios from
+ *   a. All the different groups if `group` is == '_all_',
+ *   b. Only the given group
+ *
+ * @param {String} group
+ * @return {Array}
+ */
+function buildScenariosList (group) {
+  const config = siteConfig();
```

Comment:
Can we show the total number of scenarios and also the label of scenario before running it(same as CIVIHR)?